scheduler/remote_server.py

```diff
@@ -458,8 +515,8 @@ def make_schedule():
 
             emailer.email_log(subject, log_file)
         else:
-            timeline = convert_scd_to_timeline(relevant_lines)
-            plot_path, pickle_path = plot_timeline(timeline, scd_dir, now)
+            timeline, timeline_list = convert_scd_to_timeline(relevant_lines)
```

Comment:
I think since these variables timeline and timeline_list are almost the same, we can reduce bulk by only passing out one. I think it could just be timeline_list. If you need the orders to plot (coming from the dictionary keys), move the code that converts to dictionary out of convert_scd_to_timeline and into plot_timeline (or just get the count of orders for your color calc). Then we can change timeline_to_atq to use the list instead (it's not using the keys of the dictionary). I think this should work @kkotyk ?

Reply:
I think I've got it working like this now._